lib/slim_fast/compiler.ex

```diff
@@ -9,15 +9,22 @@ defmodule SlimFast.Compiler do
 
   defp render_attribute(_, []), do: ""
   defp render_attribute(_, ""), do: ""
+  defp render_attribute(name, {:eex, opts}) do
+    value = opts[:content]
+    case value do
+      "true"  -> name
+      "false" -> ""
+      "nil"   -> ""
+      _       -> "#{name}=<%=#{value}%>"
```

Comment:
We have to wrap `<%=#{value}%>` here into `""`, so attributes values with spaces do not produce extra attributes
```elixir
assert render(~S(div ab="#{b} a" a), b: "b") == ~s(<div ab="b a">a</div>)
```